Description: Disable the "hostile" tests, they take a very long time.
Author: Stig Sandbeck Mathisen <ssm@debian.org>
Last-Update: 2012-01-21
Forwarded: no
--- gearmand.orig/tests/include.am
+++ gearmand/tests/include.am
@@ -82,21 +82,6 @@
 check_PROGRAMS+= tests/round_robin_test
 noinst_PROGRAMS+= tests/round_robin_test
 
-if TARGET_LINUX
-tests_hostileworker_test_SOURCES= tests/stress_worker.cc
-tests_hostileworker_test_SOURCES+= tests/libgearman-1.0/workers.cc
-tests_hostileworker_test_DEPENDENCIES= ${CLIENT_LDADD} libhostile/libhostile.la gearmand/gearmand
-tests_hostileworker_test_LDADD= ${CLIENT_LDADD}
-tests_hostileworker_test_LDADD+= libhostile/libhostile.la
-tests_hostileworker_test_LDADD+= $(DL_LIB)
-tests_hostileworker_test_LDADD+= $(RT_LIB)
-check_PROGRAMS+=tests/hostileworker_test
-noinst_PROGRAMS+=tests/hostileworker_test
-
-valgrind-hostileworker: tests/hostileworker_test gearmand/gearmand
-	@$(VALGRIND_COMMAND) tests/hostileworker_test
-endif
-
 # Test linking with C++ application
 tests_cpp_test_SOURCES= tests/cpp_test.cc
 tests_cpp_test_DEPENDENCIES= ${CLIENT_LDADD} gearmand/gearmand
